test/expenses.routes.test.js

```diff
@@ -768,49 +720,71 @@ describe('expenses.routes.test.js', () => {
+                  beforeEach(() => expectTwo(Transaction).tap(t => transaction = t));
 
-                it('THEN creates a transaction paid activity', () =>
-                  expectTransactionPaidActivity(group, host, transaction)
-                    .tap(activity => expect(activity.data.paymentResponse).to.be.undefined));
-              });
+                  it('THEN does not call PayPal', () => expect(payStub.called).to.be.false);
 
-              function expectTransactionCreated(expense, transaction) {
-                expect(transaction).to.have.property('netAmountInGroupCurrency', -3737);
-                expect(transaction).to.have.property('ExpenseId', expense.id);
-                // TODO remove #postmigration, info redundant with joined tables?
-                expect(transaction).to.have.property('amount', -expense.amount/100);
-                expect(transaction).to.have.property('currency', expense.currency);
-                expect(transaction).to.have.property('description', expense.title);
-                expect(transaction).to.have.property('status', 'REIMBURSED');
-                expect(transaction).to.have.property('UserId', expense.UserId);
-                expect(transaction).to.have.property('GroupId', expense.GroupId);
-                // end TODO remove #postmigration
-              }
+                  it('THEN marks expense as paid', () => expect(expense.status).to.be.equal('PAID'));
 
-              function expectTransactionPaidActivity(group, user, transaction) {
-                return Activity
-                  .findOne({ where: { type: 'group.transaction.paid' }})
-                  .tap(activity => {
-                    expect(activity.UserId).to.be.equal(user.id);
-                    expect(activity.GroupId).to.be.equal(group.id);
-                    expect(activity.TransactionId).to.be.equal(transaction.id);
-                    expect(activity.data.user.id).to.be.equal(user.id);
-                    expect(activity.data.group.id).to.be.equal(group.id);
-                    expect(activity.data.transaction.id).to.be.equal(transaction.id);
-                  });
-              }
+                  it('THEN creates transaction', () => expectTransactionCreated(expense, transaction));
+
+                  it('THEN creates a transaction paid activity', () =>
+                    expectTransactionPaidActivity(group, host, transaction)
+                      .tap(activity => expect(activity.data.paymentResponse).to.be.undefined));
+                });
+
+                function expectTransactionCreated(expense, transaction) {
+                  expect(transaction).to.have.property('netAmountInGroupCurrency', -3737);
+                  expect(transaction).to.have.property('ExpenseId', expense.id);
+                  // TODO remove #postmigration, info redundant with joined tables?
+                  expect(transaction).to.have.property('amount', -expense.amount/100);
```

Comment:
Same as above. That way, when you remove the code, tests fail and you have confirmation that you removed the right fields and all of their instances.